controllers/work-profiles.js

```diff
@@ -0,0 +1,88 @@
+const WorkProfiles = require('../models/work-profiles.js');
+
+const getAllWorkProfiles = (req, res) => {
+  WorkProfiles.find()
+  .then((WorkProfiles) => res.status(200).json({WorkProfiles}))
+  .catch((error) => res.status(500).json(error));
+};
+
+const getWorkProfileByID = (req, res) => {
+  WorkProfiles.findById(req.params.id)
+  .then((data) =>  res.json({ data }))
+  .catch((error) => res.status(404).json(error))
+};
+
+const getWorkProfilesByName = (req, res) => {
+  WorkProfiles.find({ name: req.params.name })
+  .then((matchedProfiles) => res.json({ matchedProfiles }))
+  .catch((error) => res.status(404).json(error))
+};
+
+const createWorkProfile = (req, res) => {
+  if (!req.body.name || !req.body.description) {    
+    return res.status(400).send('Please include all fields to create a new Work Profile');
+  } else {
+    const workProfile = new WorkProfiles({
+      name: req.body.name,
+      description: req.body.description,
+    });
+    workProfile.save((error, workProfile) => {
+      if (error) {
+        return res.status(500).json({ msg: 'Error while creating a new Work Profile', error });
+      }
+      return res.status(201).json(workProfile);
+    });
+  };
+}
+
+const updateWorkProfile = (req, res) => {
+  if (!req.params.id) {
+    return res.status(400).json({ msg: `You should enter an ID to be updated` });
+  } else {
+    WorkProfiles.findByIdAndUpdate(
+      req.params.id,
+      {
+        name: req.body.name,
+        description: req.body.description,
+      },
+      { new: true },
+      (error, updatedWorkProfile) => {
+        if (!updatedWorkProfile) {
+          return res.status(404).json({ msg: `Work Profile with the id: ${req.params.id} was not found` });
+        }
+        if (error) {
+          return res.status(400).json(error);
+        }
+        return res.status(200).json({ msg: 'Work Profile updated!', updatedWorkProfile });
+      }
+    );
+  }
+}
+
+const deleteWorkProfile = (req,res) =>{
+  if (!req.params.id) {
+    return res.status(400).json({ msg: `You should enter an ID to be deleted` });
+  } else {
+    WorkProfiles.findByIdAndDelete(req.params.id, (error, result) => {
+      if (error) {
+        return res.status(400).json(error);
+      } else {
+          if (result === null){
+            return res.status(404).json({ msg: 'ID not found' });
+          } else {
+            return res.status(200).json({ msg: `Work Profile with the id: ${req.params.id} was deleted!`, result });
+          }
+        }
+    });
+  }
+};
+
+module.exports = 
+  {
+    getAllWorkProfiles : getAllWorkProfiles,
```

Comment:
Let's start using `property shorthands` to asign a property any value with the same name:
```suggestion
module.exports = {
    getAllWorkProfiles,
```

Reply:
Right. Done!